onyo/commands/get.py

```diff
@@ -127,14 +127,23 @@ def get(args: argparse.Namespace, opdir: str) -> None:
     except OnyoInvalidRepoError:
         sys.exit(1)
 
-    # validate arguments
-    paths = set(Path(p) for p in args.path) or {Path('.')}
-    invalid_paths = {p for p in paths if not p.exists()}
-    paths -= invalid_paths
+    # validate path arguments
+    paths = set()
+    invalid_paths = set()
+    for path in args.path:
+        try:
+            onyo_path = repo.sanitize_path(path)
+            if not onyo_path.exists():
+                raise ValueError
+            paths.add(onyo_path)
+        except ValueError:
+            invalid_paths.add(path)
```

Comment:
This is a strange construct. Why not like this (see below)?

```suggestion
        onyo_path = repo.sanitize_path(path)
        if not onyo_path.exists():
            invalid_paths.add(path)
        else:
            paths.add(onyo_path)
```